packages/react-composites/src/composites/CallWithChatComposite/components/MoreDrawer.tsx

```diff
@@ -80,8 +80,12 @@ export const MoreDrawer = (props: MoreDrawerProps): JSX.Element => {
             : 'MoreDrawerSpeakers'
         },
         text: speaker.name,
-        onItemClick: onSelectSpeaker
-      }))
+        onItemClick: onSelectSpeaker,
+        secondaryIconProps: {
+          iconName: isDeviceSelected(speaker, props.selectedSpeaker) ? 'Accept' : undefined
```

Comment:
nit: instead of the iconName being undefined if the device isn't selected, the `secondaryIconProps` should be undefined